test/extended/cli/version.go

```diff
@@ -0,0 +1,76 @@
+package cli
+
+import (
+	"fmt"
+	"regexp"
+
+	g "github.com/onsi/ginkgo"
+	o "github.com/onsi/gomega"
+
+	"strings"
+
+	exutil "github.com/openshift/origin/test/extended/util"
+	e2e "k8s.io/kubernetes/test/e2e/framework"
+)
+
+var _ = g.Describe("[sig-cli] oc version", func() {
+	defer g.GinkgoRecover()
+
+	oc := exutil.NewCLI("oc-version")
+	ocAdmin := oc.AsAdmin()
+
+	g.It("check kubernetes version", func() {
+		podNames, err := ocAdmin.Run("get").Args("pods", "-l", "app=kube-controller-manager", "-n", "openshift-kube-controller-manager", "-o=jsonpath={.items..metadata.name}").Output()
+		o.Expect(err).NotTo(o.HaveOccurred())
+		podList := strings.Split(podNames, " ")
+
+		imageName, err := ocAdmin.Run("get").Args("pod", "-n", "openshift-kube-controller-manager", podList[0], "-o=jsonpath='{.spec.containers[0].image}'").Output()
```

Comment:
Squash above two invocations into a single one:
`oc get po -n openshift-kube-controller-manager -lapp=kube-controller-manager -o=jsonpath='{..spec.containers[0].image}'` and grab the first output as the image name

Reply:
Done